java-runtime-builder-app/src/test/java/com/google/cloud/runtimes/builder/config/AppYamlParserTest.java

```diff
@@ -60,7 +61,8 @@ private AppYaml parseFileWithContents(String contents) throws IOException {
   @Test
   public void testParse_defaultAppYaml() throws IOException {
     AppYaml result = parseFileWithContents(APP_YAML_PREAMBLE);
-    assertNull(result.getRuntimeConfig());
+    assertNotNull(result.getRuntimeConfig());
+    assertNotNull(result.getBetaSettings());
   }
 
```

Comment:
This is not sufficient for testing the new app.yaml syntax. 
I would highly recommend implementing the 2 testcases covering the true/false parsing of the flag, just to document/enforce for the future the exact syntax. In the current uncovered state, if I change the `@JsonProperty` to `enable_app_engine_apipis`, it should fail, but the tests are still passing.

Reply:
:+1: Good point! I'll add more tests.

Reply:
Added, and I caught a potential NPE in the process!